extensions/interactions/MathExpressionInput/directives/MathExpressionInputRulesService.ts

```diff
@@ -19,6 +19,22 @@
 oppia.factory('MathExpressionInputRulesService', [function() {
   return {
     IsMathematicallyEquivalentTo: function(answer, inputs) {
+      try {
+        MathExpression.fromLatex(answer.latex);
+      } catch (e) {
+        throw Error(
+          'Bad answer: ' + e.message() + ' inputs: ' +
```

Comment:
Ditto re more specific error message, and below as well.

Reply:
Done